examples/common/maths.adb

```diff
@@ -206,6 +204,45 @@ package body Maths is
     end Scaling_Matrix;
 
     --  ------------------------------------------------------------------------
+
+    function To_3 (Array_4 : Single_Array) return Single_Array is
```

Comment:
The name of this function does not tell at all what it does. Also, I do not really understand the purpose of this function. It seems like you actually want to transform a `GL.Types.Singles.Vector4_Array` to a `GL.Types.Singles.Vector3_Array`; but instead of normalizing to `W = 1`, you just drop the fourth component. But this is just guessing the intention; it seems very odd to use `Single_Array` instead of the proper vector array type for this.